stellar/README.md

```diff
@@ -68,51 +68,53 @@ stellar contract build
 Deploy the gateway contract
 
 ```bash
-node stellar/deploy-contract.js deploy axelar_gateway --chain-name <CHAIN_NAME> --wasm-path ../axelar-cgp-stellar/target/wasm32-unknown-unknown/release/stellar_axelar_gateway.optimized.wasm
+node stellar/deploy-contract.js deploy axelar_gateway --version v1.0.0
 ```
 
 Provide `--estimate-cost` to show the gas costs for the initialize transaction instead of executing it.
 
 ### Operators
 
 ```bash
-node stellar/deploy-contract.js deploy axelar_operators --chain-name <CHAIN_NAME> --wasm-path ../axelar-cgp-stellar/target/wasm32-unknown-unknown/release/stellar_axelar_operators.optimized.wasm
+node stellar/deploy-contract.js deploy axelar_operators --version v1.0.0
 ```
 
 ### Gas Service
 
 ```bash
-node stellar/deploy-contract.js deploy axelar_gas_service --chain-name <CHAIN_NAME> --wasm-path ../axelar-cgp-stellar/target/wasm32-unknown-unknown/release/stellar_axelar_gas_service.optimized.wasm
+node stellar/deploy-contract.js deploy axelar_gas_service --version v1.0.0
 ```
 
 ### Interchain Token Service
 
 Deploy Interchain Token and Token Manager wasm first.
 
 ```bash
-node stellar/deploy-contract.js deploy interchain_token --chain-name <CHAIN_NAME> --wasm-path ../axelar-cgp-stellar/target/wasm32-unknown-unknown/release/stellar_interchain_token.optimized.wasm
-node stellar/deploy-contract.js deploy token_manager --chain-name <CHAIN_NAME> --wasm-path ../axelar-cgp-stellar/target/wasm32-unknown-unknown/release/stellar_token_manager.optimized.wasm
-node stellar/deploy-contract.js deploy interchain_token_service --chain-name <CHAIN_NAME> --wasm-path ../axelar-cgp-stellar/target/wasm32-unknown-unknown/release/stellar_interchain_token_service.optimized.wasm
+node stellar/deploy-contract.js deploy interchain_token --version v1.0.0
+node stellar/deploy-contract.js deploy token_manager --version v1.0.0
+node stellar/deploy-contract.js deploy interchain_token_service --version v1.0.0
 ```
 
 ### Example
 
+Note that example contract should use `--wasm-path` option to deploy contract
```

Comment:
Why is this needed? the example wasm should be pushed to R2 as well, just not crates.io. Is that not the case?